packages/shared-auth/mode/stack-auth.ts

```diff
@@ -0,0 +1,85 @@
+import { bearer } from "@elysiajs/bearer";
+import { Elysia, status } from "elysia";
+import { createRemoteJWKSet, jwtVerify } from "jose";
+
+import { projectId, secretServerKey } from "../env";
+
+const jwks = createRemoteJWKSet(
+  new URL(
+    `https://api.stack-auth.com/api/v1/projects/${projectId}/.well-known/jwks.json`,
+  ),
+);
+
+const accessToken = () =>
+  new Elysia({ name: "access-token" }).derive(
+    { as: "global" },
+    function deriveAccessToken({ request }) {
```

Comment:
Redeclaring a single variable to a different name and using derive for a variable that is only used once. Feels unnecessary boilerplate.